src/utils/rest-api.js

```diff
@@ -261,6 +261,20 @@ export function deleteElement(elementUuid) {
     });
 }
 
+export function deleteElements(elementUuids, activeDirectory) {
```

Comment:
```suggestion
export const deleteElements = (elementUuids, activeDirectory) => {
```

Reply:
All functions in rest-api are declared as functions. is there a reason to change it ?